modal/image.py

```diff
@@ -1218,6 +1218,23 @@ def env(self, vars: Dict[str, str]) -> "_Image":
             dockerfile_commands=["FROM base"] + [f"ENV {key}={shlex.quote(val)}" for (key, val) in vars.items()]
         )
 
+    @typechecked
+    def workdir(self, path: str) -> "_Image":
+        """Sets the working directory for subequent image build steps.
+
+        **Example**
+
+        ```python
+        image = (
+            modal.Image.debian_slim()
+                .run_commands("git clone https://xyz app")
+                .workdir("/app")
+                .run_commands("yarn install")
+        )
+        ```
+        """
+        return self.extend(dockerfile_commands=["FROM base"] + [f"WORKDIR {shlex.quote(path)}"])
```

Comment:
will this change the workdir for subsequent steps as well?

Reply:
Yes, see https://github.com/modal-labs/modal/pull/6976